packages/contracts-bedrock/src/dispute/FaultDisputeGame.sol

```diff
@@ -475,7 +478,7 @@ contract FaultDisputeGame is IFaultDisputeGame, Clone, ISemver {
     }
 
     /// @inheritdoc IFaultDisputeGame
-    function resolveClaim(uint256 _claimIndex) external {
+    function resolveClaim(uint256 _claimIndex, uint256 _numToResolve) external {
```

Comment:
I think we should update the doc comments on the `IFaultDisputeGame.resolveClaim` method to say that if the `_numToResolve` argument is `0`, then it will check all child subgames.